src/foam/mlang/MLang.java

```diff
@@ -58,6 +58,17 @@ public static Predicate AND(Predicate... args) {
     return new And(args);
   }
 
+  public static Sink GROUPBY(Object o1, Object o2) {
```

Comment:
Rename to GROUP_BY to be compatible with the JS version.

Reply:
Ok.